Sources/CardPayments/CardClientError.swift

```diff
@@ -28,6 +28,15 @@ enum CardClientError {
 
         /// 6. GraphQLClient is unexpectedly nil
         case nilGraphQLClientError
+        
+        /// 7. An error from 3DS verification
+        case threeDSVerificationError
+        
+        /// 8. Missing Deeplink URL from 3DS
+        case missingDeeplinkURLError
+        
+        /// 9. Missing Deeplink URL from 3DS
```

Comment:
It looks like this docstring is a copy of the above, should it be different?

Reply:
Good catch!

Reply:
Thanks, done: https://github.com/paypal/paypal-ios/pull/232/commits/2896471c1533de72880afd18545a33a374f9cfc0